### DISCUSSION TITLE
Coder OSS V2 Upgrade

### DISCUSSION THREAD

**Discussion by vl00920760** [coder/coder#6720] [Announcements]
Hi, I am upgrading coder oss v2 from 0.18.1 to a higher version.
The platform is a Kubernetes environment. One can consider as a Azure AKS.

The installation with following documentation installs only the latest version. https://coder.com/docs/v2/latest/install/kubernetes#upgrading-coder-via-helm

Referring to the Coder version 1 document, I can choose the version from an asset section. example - helm install coder ./coder_helm_0.18.2.tgz -n coder

By using above option, my pod stuck in Crashbackloopoff. 

As a result, I am not getting desired result. Can anyone help me here please ?

Thanks, 

Vivek

**Comment by matifali** [comment]
Coder v1 and v2 are different products. You should be able to update without any issue for V2

**Reply by vl00920760** [reply]
I completed this upgrade without any issue. This is using Helm upgrade.
